src/prefect_server/api/runs.py

```diff
@@ -160,7 +160,10 @@ async def create_flow_run(
             run_config = flow.run_config
 
     # check parameters
-    run_parameters = flow.flow_group.default_parameters
+    run_parameters = {
+        param["name"]: param["default"] for param in flow.parameters if param["default"]
+    }
+    run_parameters.update(flow.flow_group.default_parameters or {})
```

Comment:
should the flow group parameters take precedence over the flow parameters here? it seems like the flow's params would take priority

Reply:
👍 I'd preserve whatever order is there today

Reply:
okay yep I see that order in a test case, lgtm